vllm/model_executor/layers/quantization/kv_cache.py

```diff
@@ -0,0 +1,66 @@
+from vllm.model_executor.layers.quantization.base_config import QuantizeMethodBase, QuantizationConfig
+import torch
+from vllm.utils import print_warning_once
+
+
+class BaseKVCacheMethod(QuantizeMethodBase):
+
```

Comment:
This class could use a doc string describing the assumption of accepting either a single kv_scale or separate k_scale+v_scale and ending up with a single kv_scale